shell/ci/testing/coverage.sh

```diff
@@ -29,16 +29,12 @@ if [[ -z $file ]]; then
 fi
 
 coverage_provider=$(yq -r '.arguments.coverage.provider' <"$(get_service_yaml)" 2>/dev/null)
-if [[ -z $coverage_provider ]] || [[ $coverage_provider == "null" ]]; then
-  info "No coverage provider configured (.arguments.coverage) is empty"
-  exit 0
-fi
 
-if [[ $coverage_provider == "codecov" ]]; then
-  "$DIR/codecov/upload-coverage.sh" "$file" "$group"
-elif [[ $coverage_provider == "coveralls" ]]; then
-  "$DIR/coveralls/upload-coverage.sh" "$file" "$group"
-else
-  error "Unknown coverage provider \"$coverage_provider\", skipping coverage upload"
-  exit 0
-fi
+case $coverage_provider in
+"coverbot")
+  exec "$DIR/coverbot/upload-coverage.sh" "$file" "$group"
+  ;;
+*)
```

Comment:
Should we keep `case` statements for `coveralls` and `codecov` so that they are still functional?

Reply:
Yes, probably. Gonna re-write the original handling implementation for those providers (which was actually via if/else statements) as case statements.